game/src/client/state/positions.ts

```diff
@@ -7,16 +7,52 @@ export const playerPosition = {
   targetY: 0,
   angle: 0,
 };
-
-export const npcPosition = {
+export type EntityType = 'robot' | 'alien';
+export interface NPCPositions{
```

Comment:
`NPCPositions` should be singular: `NPCPosition`.

Also, it's no longer accurate to call this just "position", as it comprises other aspects of the NPC. A more appropriate name would be `NPCConfig`.